docs/guides/setup-dbt.mdx

```diff
@@ -1,257 +1,103 @@
+- Wrote the data to a Postgres source
 
-   > [dbt sources](https://docs.getdbt.com/docs/build/sources)
-   >
-   > When a dbt model depends on an upstream block that isn’t a dbt model, a
-   > source for that block is automatically added to the
-   > `demo_project/dbt/demo/models/example/mage_sources.yml` file.
-   >
-   > Read more about dbt sources in their
-   > [documentation](https://docs.getdbt.com/docs/build/sources).
+To see the output, you can use a querying tool (like [DataGrip](https://www.jetbrains.com/datagrip/) or [psql](https://www.postgresql.org/docs/current/app-psql.html)) to the locally hosted Postgres database and query `public.analytics.cur_customers`.
 
-3. Run the dbt model block by pressing the play button on the top right of the
-   block or by pressing `Command` + `Enter`.
-4. You should see a preview of the query execution logs. To see the query results,
-   click the `Expand table` link at the bottom right corner.
+| customer_id | first_name | last_name | letters_first_name | is_alliterative |
+| ----------- | ---------- | --------- | ------------------ | --------------- |
+| 1           | Michael    | P.        | 7                  | false           |
+| 2           | Shawn      | M.        | 5                  | false           |
+| 3           | Kathleen   | P.        | 8                  | false           |
+| 4           | Jimmy      | C.        | 5                  | false           |
+| 5           | Katherine  | R.        | 9                  | false           |
+| 6           | Sarah      | R.        | 5                  | false           |
+| 7           | Martin     | M.        | 6                  | true            |
+| 8           | Frank      | R.        | 5                  | false           |
+| 9           | Jennifer   | F.        | 8                  | false           |
+| 10          | Henry      | W.        | 5                  | false           |
 
----
-
-## 7. Add test for dbt model
-
-1. On the right side of the screen, click the tab labeled **`Terminal`**.
-2. Create a new dbt test file by running the following command:
-   ```bash
-   touch demo_project/dbt/demo/tests/test_my_second_dbt_model.sql
-   ```
-3. On the left side of the page in the file browser, expand the folder
-   `demo_project/dbt/demo/tests/` and click the file named
-   `test_my_second_dbt_model.sql`. If you don’t see it, refresh the page.
-4. Paste the following SQL in the file:
-   ```sql
-   SELECT id
-   FROM {{ ref('my_second_dbt_model') }}
-   GROUP BY id
-   HAVING (id = 0)
-   ```
-5. Read more about dbt tests in their
-   [documentation](https://docs.getdbt.com/docs/build/tests).
-
----
+</Accordion>
+<Accordion title="Execute dbt from an external repo" icon="pipe-circle-check">
 
-## 8. Execute pipeline end-to-end
+Select the `dynamic_dbt_pipeline` by double-clicking— this will take you directly to the editor. You can also single click, then select the "code" icon from the side nav.
```

Comment:
I think this could be clearer by saying "double-clicking the row from the list of pipelines".

Reply:
Perhaps rephrase `then select the "code" icon from the side nav.` to `then select "Edit pipeline" (with the "code" icon) from the side nav`.